com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/ResolveConfigurationEntriesStep.java

```diff
@@ -0,0 +1,60 @@
+package com.sap.cloud.lm.sl.cf.process.steps;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import com.sap.cloud.lm.sl.cf.client.lib.domain.CloudApplicationExtended;
+import com.sap.cloud.lm.sl.cf.core.cf.v1_0.ConfigurationEntriesCloudModelBuilder;
+import com.sap.cloud.lm.sl.cf.core.model.ConfigurationEntry;
+import com.sap.cloud.lm.sl.cf.core.model.DeployedMta;
+import com.sap.cloud.lm.sl.cf.core.model.DeployedMtaModule;
+import com.sap.cloud.lm.sl.cf.core.util.CloudModelBuilderUtil;
+import com.sap.cloud.lm.sl.cf.process.message.Messages;
+import com.sap.cloud.lm.sl.common.SLException;
+import com.sap.cloud.lm.sl.mta.model.v1_0.DeploymentDescriptor;
+
+@Component("resolveConfigurationEntriesStep")
+@Scope(BeanDefinition.SCOPE_PROTOTYPE)
+public class ResolveConfigurationEntriesStep extends BuildCloudDeployModelStep {
```

Comment:
I suppose the logic here is taken from the CloudModelBuilderStep?
Where this step is being used? If it is called each time, why not remove the logic from the CloudModelBuilder?

Reply:
It is changed in the next commit